### PR TITLE
Enable check for bare 'except' blocks (E722)

### PR DESCRIPTION
**Pull request by winniehell**
## Description of the issue/feature this PR addresses

Enables flake8 check for bare 'except' blocks (E722) and fixes existing violations.

### CODE REVIEW

**Comment by winniehell** [comment]
catching `Exception` is still very broad but finding the right exception classes for all existing violations would take a lot of effort

File: src/bika/lims/content/analysisrequest.py [hunk 1]
```diff
@@ -1831,7 +1831,7 @@ def getVerifier(self):
         # noinspection PyBroadException
```
Comment (winniehell) [inline src/bika/lims/content/analysisrequest.py:1831]:
```suggestion
```

File: src/bika/lims/content/analysisrequest.py [hunk 2]
```diff
@@ -1831,7 +1831,7 @@ def getVerifier(self):
...
         try:
             review_history = wtool.getInfoFor(self, 'review_history')
-        except:  # noqa FIXME: remove blind except!
+        except Exception:  # noqa FIXME: remove blind except!
```
Comment (winniehell) [inline src/bika/lims/content/analysisrequest.py:1834]:
```suggestion
        except Exception:
```

File: src/bika/lims/content/analysisrequest.py [hunk 3]
```diff
@@ -2117,7 +2117,7 @@ def getRejecter(self):
         # noinspection PyBroadException
```
Comment (winniehell) [inline src/bika/lims/content/analysisrequest.py:2117]:
```suggestion
```

File: src/bika/lims/content/analysisrequest.py [hunk 4]
```diff
@@ -2117,7 +2117,7 @@ def getRejecter(self):
...
         try:
             review_history = wtool.getInfoFor(self, 'review_history')
-        except:  # noqa FIXME: remove blind except!
+        except Exception:  # noqa FIXME: remove blind except!
```
Comment (winniehell) [inline src/bika/lims/content/analysisrequest.py:2120]:
```suggestion
        except Exception:
```

File: src/bika/lims/content/analysisrequest.py (hunk 1 above)
Comment (winniehell) [inline src/bika/lims/content/analysisrequest.py:1831]:
```suggestion
```

**Review by ramonski — APPROVED** [review]

**Pull request merged**